spring-cloud-generator/scripts/generate-library-list.sh

```diff
@@ -66,24 +55,13 @@ for d in ./google-cloud-java/*java-*/; do
     continue
   fi
 
-  # get monorepo-name as pattern ./google-cloud-java/<monorepo_name>/
-  monorepo_name=$(echo $d | sed 's#^./google-cloud-java/##' | sed 's#/$##')
+  # parse proto path from generation_config.yaml, find by api_shortname
+  # then sort and keep latest stable version
+  library=$(yq -r '.libraries[] | select(.api_shortname == "'"$api_shortname"'")' ./google-cloud-java/generation_config.yaml)
+  proto_paths_stable=$(echo "$library" | yq -r '.GAPICs[] | select(.proto_path | test("/v[0-9]+$")) | .proto_path')
```

Comment:
What happened if no stable proto path exists in a library?

Reply:
We only generate spring starter for stable libraries ([logic](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/9f26aedc2877d304fbcc45e78c149dee7ca51e4e/spring-cloud-generator/scripts/generate-library-list.sh#L48-L51)).
I would expect if a library is stable (according to repo-metadata.json), it should have at least one stable (not alpha/beta) proto path. Correct me if I am wrong?
Btw, this logic is the same prior to this change, only parsing proto path directly from "OwlBot.yaml"

Reply:
@JoeWang1127 following up on this, let me know if the above answers your question? If so, I am going to merge this in.